src/SDKs/Batch/DataPlane/Azure.Batch/Azure.Batch.csproj

```diff
@@ -10,7 +10,7 @@
     <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
     <PackageId>Microsoft.Azure.Batch</PackageId>
     <Description>This client library provides access to the Microsoft Azure Batch service.</Description>
-    <Version>9.0.1</Version>
+    <Version>10.0.0</Version>
```

Comment:
@matthchr why a major version bump?
Can you also release notes?
Is the major version due to breaking changes or underlying API change?

Reply:
I did release notes, they're included in the PR.
There are breaking changes both in the SDK and in the underlying API, which is why it's a major version bump. The release notes outlines what changed and what breaks there were.